### PR TITLE
add Ascend desktop and VDIs

### PR DESCRIPTION
**Pull request by johrstrom**
add Ascend desktop and VDIs.  Note this also has some bug fixes like
* broken links
* cores should be a number_field
* renamed VDIs to Lightweight Desktop

### CODE REVIEW

File: apps.awesim.org/apps/bc_desktop/ascend.yml.erb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+      [slurm migration]: https://www.osc.edu/resources/technical_support/supercomputers/owens/environment_changes_in_slurm_migration
+    pattern: '^([\w]+@osc:\d+,{0,1})+$'
+  node_type:
+    widget: select
+    label: "Node type"
+    help: |
+      Need docs
+    options:
+      - [ "any", "any" ]
```
Comment (treydock) [inline apps.awesim.org/apps/bc_desktop/ascend.yml.erb:72]: I wonder if it's better to offer CPU vs GPU.  The "Any" (ie not asking for GPU) will get CPU only job where max cores is 84.

File: apps.awesim.org/apps/bc_desktop/ascend.yml.erb [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    widget: select
+    options:
+      <%- groups.each do |group| %>
+      - "<%= group %>"
+      <%- end %>
+  cores:
+    widget: number_field
+    value: 46
+    min: 1
+    max: 88
```
Comment (treydock) [inline apps.awesim.org/apps/bc_desktop/ascend.yml.erb:56]: If the Ascend desktop runs on login nodes, should not allow this many cores, in fact the cgroup on login node will be limited to about 7 cores, which is something we set on all login node sessions via PAM.

Comment (johrstrom) [reply apps.awesim.org/apps/bc_desktop/ascend.yml.erb:56]: Desktops run through Slurm. `Lightweight Desktops` (`VDIs`) are on quick & ascend-login and I don't believe they have cores as an option.

File: apps.awesim.org/apps/bc_desktop/ascend.yml.erb (hunk 1 above)
Comment (johrstrom) [reply apps.awesim.org/apps/bc_desktop/ascend.yml.erb]: Done!

**Pull request merged**